### ISSUE TITLE
Train loss is nan

### ISSUE THREAD

**Issue by harishyadav1465** [mleimeister/ctc_tensorflow_voxforge#2]
i have learned and compiled your code but i am getting train_loss=nan,val_loss=nan

**Comment by mleimeister** [comment]
Hi, sorry for my late answer. I haven't touched the code in a year and it might not be compatible anymore with current tf versions. Can you send me which Python and Tensorflow versions you are using? And the command line output you're getting. Then I can look into it.
